docs/diagrams/ImportClassDiagram.puml

```diff
@@ -0,0 +1,33 @@
+@startuml
+class ImportCommandParser implements Parser {
```

Comment:
There might be some missing navigability.
You might want to check whether a 'T' text box image is supposed to appear next to your interface.
The names of your attributes could be easier to comprehend, for example 'OT_RETURN' and 'OT_ORDER'.